test/TestProjects/MgmtScopeResource/Generated/PolicyAssignmentContainer.cs

```diff
@@ -270,207 +269,343 @@ public async virtual Task<Response<bool>> CheckIfExistsAsync(string policyAssign
+                    scope.Start();
+                    try
+                    {
+                        var response = await _policyAssignmentsRestClient.GetForResourceAsync(Id.ResourceGroupName, Id.ResourceType.Namespace, Id.Parent.GetParts(1), Id.ResourceType.Types.Last(), Id.Name, filter, top, cancellationToken: cancellationToken).ConfigureAwait(false);
+                        return Page.FromValues(response.Value.Value.Select(value => new PolicyAssignment(Parent, value)), response.Value.NextLink, response.GetRawResponse());
+                    }
+                    catch (Exception e)
+                    {
+                        scope.Failed(e);
+                        throw;
+                    }
                 }
-                catch (Exception e)
+                async Task<Page<PolicyAssignment>> NextPageFunc(string nextLink, int? pageSizeHint)
                 {
-                    scope.Failed(e);
-                    throw;
+                    using var scope = _clientDiagnostics.CreateScope("PolicyAssignmentContainer.GetAll");
+                    scope.Start();
+                    try
+                    {
+                        var response = await _policyAssignmentsRestClient.GetForResourceNextPageAsync(nextLink, Id.ResourceGroupName, Id.ResourceType.Namespace, Id.Parent.GetParts(1), Id.ResourceType.Types.Last(), Id.Name, filter, top, cancellationToken: cancellationToken).ConfigureAwait(false);
+                        return Page.FromValues(response.Value.Value.Select(value => new PolicyAssignment(Parent, value)), response.Value.NextLink, response.GetRawResponse());
+                    }
+                    catch (Exception e)
+                    {
+                        scope.Failed(e);
+                        throw;
+                    }
                 }
+                return PageableHelpers.CreateAsyncEnumerable(FirstPageFunc, NextPageFunc);
+            }
+        }
+
+        /// <summary> Filters the list of <see cref="PolicyAssignment" /> for this resource group represented as generic resources. </summary>
+        /// <param name="nameFilter"> The filter used in this operation. </param>
+        /// <param name="expand"> Comma-separated list of additional properties to be included in the response. Valid values include `createdTime`, `changedTime` and `provisioningState`. </param>
+        /// <param name="top"> The number of results to return. </param>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        /// <returns> A collection of resource that may take multiple service requests to iterate over. </returns>
+        public virtual Pageable<GenericResource> GetAllAsGenericResources(string nameFilter, string expand = null, int? top = null, CancellationToken cancellationToken = default)
```

Comment:
This method only applies for resource group and subscription.

Reply:
This has been fixed